Binary files wxorig/include/.DS_Store and wxnew/include/.DS_Store differ
diff -uNr wxorig/include/wx/bookctrl.h wxnew/include/wx/bookctrl.h
--- wxorig/include/wx/bookctrl.h	2010-07-18 18:54:00.000000000 -0400
+++ wxnew/include/wx/bookctrl.h	2010-10-18 11:29:22.000000000 -0400
@@ -321,7 +321,7 @@
     void OnSize(wxSizeEvent& event);
 
     // controller buddy if available, NULL otherwise (usually for native book controls like wxNotebook)
-    wxControl *m_bookctrl;
+    wxWindow *m_bookctrl;
 
     // Whether to shrink to fit current page
     bool m_fitToCurrentPage;
